go/internal/feast/server/http_server.go

```diff
@@ -203,6 +226,15 @@ func (s *httpServer) getOnlineFeatures(w http.ResponseWriter, r *http.Request) {
 	}
 	entitiesProto := make(map[string]*prototypes.RepeatedValue)
 	for key, value := range request.Entities {
+		var entity, err = s.fs.GetEntityByJoinKey(key)
```

Comment:
request.Entities will also have information about request source features in the context of ODFV's. If fs.GetEntityByJoinKey() raises exception, it will fail all ODFV use cases.

Reply:
Hmm I hadn't thought about ODFVs. Let me take a look at it again.

Reply:
@wparsley-eg Can you confirm if we get the ODFV's request data in entities or request_context?